src/Features/Core/Portable/Completion/CommonCompletionItem.cs

```diff
@@ -1,14 +1,32 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System;
 using System.Collections.Immutable;
 using System.Linq;
 using Microsoft.CodeAnalysis.Tags;
 using Roslyn.Utilities;
 
 namespace Microsoft.CodeAnalysis.Completion
 {
-    internal static class CommonCompletionItem 
+    internal static class CommonCompletionItem
     {
+        [Obsolete("This is a compatibility shim for FSharp; please do not use it.", error: true)]
+        public static CompletionItem Create(
```

Comment:
@dpoeschl Didn't we already have some other workaround for this? Or am I confusing this with the signature help thing?

Reply:
You're probably thinking of [the SigHelp one](https://github.com/dotnet/roslyn/pull/30844).